addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -87,7 +91,7 @@
 
     // RIBBON: Kanban specific
     // Ungrouped specific
-    .o_survey_kanban_card_ungrouped {
+    .o_survey_kanban_card > div:first-child {
```

Comment:
Actually is that really necessary? Do we have more than one div that has the class "ribbon"?